.github/workflows/docs-ci.yml

```diff
@@ -39,11 +39,11 @@ jobs:
           .github/scripts/gradle-properties.sh >> gradle.properties
           cat gradle.properties
 
-      - name: All Javadoc
+      - name: All Javadoc, ts/js docs
         uses: burrunan/gradle-cache-action@v1
         with:
           job-id: allJavadoc
-          arguments: --scan combined-javadoc:allJavadoc
+          arguments: --scan combined-javadoc:allJavadoc :web-client-api:types:typedoc
```

Comment:
So far, we've separated the different documentation parts into their own jobs. Is there a reason we don't want to do that w/ ts/js?

Reply:
Historically, it was just a decision I made to parallelize the docs building process for what I felt where distinct builds. I think there is some value in seeing them fail independently of one another. I think it makes sense to continue the distinct jobs pattern _or_ combine them all into a single job.

Reply:
Ok, its a separate job now.